plugins/jetty/src/main/java/com/navercorp/pinpoint/plugin/jetty/JettyPlugin.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2014 NAVER Corp.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.navercorp.pinpoint.plugin.jetty;
+
+import com.navercorp.pinpoint.bootstrap.plugin.ProfilerPlugin;
+import com.navercorp.pinpoint.bootstrap.plugin.ProfilerPluginContext;
+import com.navercorp.pinpoint.bootstrap.plugin.transformer.ClassFileTransformerBuilder;
+import com.navercorp.pinpoint.plugin.jetty.JettyDetector;
+import com.navercorp.pinpoint.plugin.jetty.JettyConfiguration;
+
+public class JettyPlugin implements ProfilerPlugin, JettyConstants{
+
+	@Override
+	public void setup(ProfilerPluginContext context) {
+		context.addApplicationTypeDetector(new JettyDetector());
+        JettyConfiguration config = new JettyConfiguration(context.getConfig());
+		addServerInterceptors(context, config);
+	}
+
+    private void addServerInterceptors(ProfilerPluginContext context, JettyConfiguration config){
+    	ClassFileTransformerBuilder builder = context.getClassFileTransformerBuilder("org.eclipse.jetty.server.Server");
+        builder.injectInterceptor("com.navercorp.pinpoint.plugin.jetty.interceptor.ServerHandleInterceptor", config.getJettyExcludeUrlFilter());
+        //builder.injectInterceptor("com.navercorp.pinpoint.plugin.jetty.interceptor.ServerHandleAsyncInterceptor");
+        builder.injectMetadata(METADATA_ASYNC_TRACE_ID);
```

Comment:
This is to trace handleAsync(), right? If so, injecting anything related to a request into Server object is not a good idea because it handles many requests concurrently.